trunk/SUAVE/Methods/Weights/Buildups/eHelicopter/prop.py

```diff
@@ -0,0 +1,225 @@
+# prop.py
+#
+# Created: Jun 2017, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from SUAVE.Core import Units
+from SUAVE.Attributes import Solids
+import numpy as np
+
+#-------------------------------------------------------------------------------
+# Prop
+#-------------------------------------------------------------------------------
+
+def prop(rProp, maxThrust, nBlades):
+    """weight = SUAVE.Methods.Weights.Correlations.eHelicopter.prop(
+            rProp,
+            maxThrust,
+            nBlades
+        )
+
+        Calculates propeller blade pass for an eVTOL vehicle based on assumption
+        of a NACA airfoil prop, an assumed cm/cl, tip Mach limit, and structural
+        geometry.
+
+        Intended for use with the following SUAVE vehicle types, but may be used
+        elsewhere:
+
+            eHelicopter
+            eTiltwing
+            eTiltrotor
+            eStopped_Rotor
+
+        Originally written as part of an AA 290 project inteded for trade study
+        of the above vehicle types.
+
+        Inputs:
+
+            rProp:      Propeller Radius            [m]
+            maxThrust:  Maximum Motor Thrust        [N]
+            nBlades:    Number of Propeller Blades  [Unitless]
+
+        Outputs:
+
+            weight:     Propeller Mass              [kg]
+    """
+
+    chord       = rProp * 0.1    # Assumed Prop chord
+    N           = 5              # Number of Analysis Points
+    SF          = 1.5            # Saftey Factor
+    toc         = 0.12           # Average Blade t/c
+    fwdWeb      = [0.25, 0.35]   # Forward Web Chord Fraction locations
+    xShear      = 0.25           # Approximate Shear Center
+    rootLength  = rProp * 0.1    # Assumed Root Fitting Length
+    grace       = 1.2            # Grace Factor for Estimation
+    sound       = 340.294        # Assumed Speed of Sound
+    tipMach     = 0.65           # Propeller Tip Mach Number Limit
+    cmocl       = 0.02           # Assumed cm/cl for Sizing Torsion
+
+#-------------------------------------------------------------------------------
+# Airfoil
+#-------------------------------------------------------------------------------
+
+    NACA = np.multiply(5 * toc, [0.2969, -0.1260, -0.3516, 0.2843, -0.1015])
```

Comment:
I wish you asked me about all of this. I have all this stuff figured out for you. Ask me in the meeting tomorrow.